pkgs/os-specific/linux/rtl8812au/default.nix

```diff
@@ -42,9 +42,10 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Driver for Realtek 802.11ac, rtl8812au, provides the 8812au mod";
-    homepage = "https://github.com/gordboy/rtl8812au-5.6.4.2";
-    license = licenses.gpl2;
+    homepage = "https://github.com/gordboy/rtl8812au-5.9.3.2";
+    license = licenses.gpl2Only;
     platforms = platforms.linux;
     maintainers = with maintainers; [ danielfullmer ];
+    broken = kernel.kernelOlder "4.10" || kernel.isHardened;
```

Comment:
Why is this marked broken for kernels older than 4.10?  The github page even says it was built and tested on kernels as old as 5.4.

Reply:
4_4 and 4_9 were failing

Reply:
5.4 is newer than both of those

Reply:
Whoops. I looked at `4.10` but I read `5.10`.